.golangci.yml

```
@@ -0,0 +1,88 @@
+linters:
```

[nit] since you're enabling `misspell` let's configure it to american english
```suggestion
linters-settings:
  misspell:
    locale: US
linters:
```